doc/u-boot-env.md

```diff
@@ -0,0 +1,143 @@
+# Boot storage device setting
+
+Boot device is selected by setting 'bootcmd' variable.
+Possible settings for available boot devices are:
+```
+emmc: setenv bootcmd run bootcmd_emmc
+sd0:  setenv bootcmd run bootcmd_mmc0
+sd3:  setenv bootcmd run bootcmd_mmc1
+net:  setenv bootcmd run bootcmd_tftp
+```
+
+## Salvator-X/-XS and H3ULCB
+```
+setenv bootdelay 3
+setenv baudrate 115200
+setenv bootargs
+setenv ethact ravb
+setenv ipaddr 192.168.1.10
+setenv serverip 192.168.1.100
+setenv initrd_high 0xffffffffffffffff
+```
+
+If you need to set MAC, use variable `ethaddr` and MAC address provided on sticker on ethernet connector on board.
+Use following format:
+```
+setenv ethaddr 12:34:56:78:9A:BC
+```
+
+## TFTP boot
+```
+setenv bootcmd run bootcmd_tftp
+setenv bootcmd_tftp 'run tftp_xen_load; run tftp_dtb_load; run tftp_kernel_load; run tftp_xenpolicy_load; run tftp_initramfs_load; bootm 0x48080000 0x84000000 0x48000000'
```

Comment:
In this and in the following sections I'd rather change order of variables. Now you are referring to a commands that are not available yet, which makes it hard to parse this in a head.

I think that `bootcmd_tftp` should be introduced after `tftp_xen_load`, `tftp_dtb_load`,.. etc, and `bootcmd` - after `bootcmd_tftp`.

Reply:
Thanks for pointing. Fixed according to comment.